src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -995,38 +995,6 @@ async void M(CancellationToken ct)
             return VerifyCS.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
-        [Fact]
-        public Task CS_Diagnostic_OverloadToken_WithConfigureAwait()
```

Comment:
Duplicates `CS_Diagnostic_OverloadToken`

Reply:
Instead of deleting this test, let's update it to match the test name (which should have been the original intent) and add `.ConfigureAwait(..)`

Reply:
Makes sense! I will try to update.

Reply:
+1